src/components/table/OcTableFiles.vue

```diff
@@ -681,391 +710,4 @@ export default {
   }
 </script>
 ```
-
-## Shared with others files table
```

Comment:
Why delete all the examples? I found them quite useful in the past:)

Reply:
because i'm an idiot, i bring it back. Thanks for pointing out